tests/PKSim.Tests/IntegrationTests/IndividualMapperSpecs.cs

```diff
@@ -0,0 +1,95 @@
+﻿using OSPSuite.BDDHelper;
+using OSPSuite.BDDHelper.Extensions;
+using OSPSuite.Core.Domain;
+using PKSim.Core.Model;
+
+namespace PKSim.IntegrationTests
+{
+   public class When_loading_a_snapshot_file_containing_expression_created_in_v10 : ContextWithLoadedSnapshot
+   {
+      private Individual _individual;
+      private ExpressionProfile _expressionProfileEnzyme;
+      private ExpressionProfile _expressionProfileTransporter;
+
+      public override void GlobalContext()
+      {
+         base.GlobalContext();
+         LoadSnapshot("ind_expression_v10");
```

Comment:
@Yuri05 WHy not have integration tests for snapshot too!!

Reply:
Theoretically, I should be able to run the installation validator and this should all be the same as in v10 if this is really working...